submission_rules.adoc

```diff
@@ -472,14 +472,14 @@ This requirement applies even to Open submissions, though the aspects do not nee
 
 ### Compliance Testing
 
-Submitters must run the compliance tests to verify that their submission achieves a basic level of compliance with a subset of the MLPerf rules. If compliance testing identifies a potential issue with the submission, the onus is on the submitter to provide an adequate explanation to the working group.
-
 #### Training
 
 This section in progress [TODO].
 
 #### Inference
 
+Submitters must run the compliance tests for their closed division submissions to verify that their submission achieves a basic level of compliance with a subset of the MLPerf rules. If compliance testing identifies a potential issue with the submission, the onus is on the submitter to provide an adequate explanation to the working group.
```

Comment:
Do we want to say "working group" or "review committee" here?

Reply:
good catch